### PR TITLE
added skeleton files

### PR DESCRIPTION
**Pull request by Wyrd00**
relates #2

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+
```
Comment (matthewdking) [inline src/app.js]: add the other stuff from the workshop
app.set and app.use

**Review by sajedazoabi — APPROVED** [review]

**Review by matthewdking — APPROVED** [review]

**Review by matthewdking — CHANGES_REQUESTED** [review]
solve the merge conflicts please please please

**Review by matthewdking — APPROVED** [review]

File: src/tests/router.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const tape = require('tape');
+const supertest = require('supertest');
+//add route here
+
+tape('I work I work?', (t) => {
+	let num = 3.14;
+	t.deepEqual(num, 3.14, "NomNomNom");
```
Comment (Karyum) [inline src/tests/router.test.js:7]: typical Amy test

**Review by Karyum — APPROVED** [review]
:+1: great job

**Pull request merged**